src/link_rfu_2.c

```diff
@@ -433,38 +437,38 @@ static void sub_80F8B34(u8 taskId)
         }
         break;
     case 14:
-        sub_80FD760(0);
-        Rfu.unk_04 = 15;
+        rfu_LMAN_stopManager(0);
+        Rfu.state = 15;
         break;
     case 15:
         break;
     case 16:
         Rfu.unk_cdb = 0;
-        sub_80FEA10(sub_80F8DA8);
-        sub_80FAFE0(1);
+        rfu_LMAN_setMSCCallback(MSCCallback_SetUnkCDB);
+        UpdateGameData_GroupLockedIn(1);
         sub_80F8AA4();
         sub_80F8AEC();
-        Rfu.unk_04 = 20;
+        Rfu.state = 20;
         gTasks[taskId].data[1] = 8;
-        Rfu.unk_0c = 1;
+        Rfu.parent_child = MODE_PARENT;
         CreateTask(sub_80FA834, 5);
         Rfu.unk_ce8 = TRUE;
         DestroyTask(taskId);
         break;
     }
 }
 
-void sub_80F8CFC(void)
+void LinkRfu_CreateConnectionAsParent(void)
 {
-    sub_80FD538(1, 0, 240, gUnknown_843EC8C);
+    rfu_LMAN_establishConnection(MODE_PARENT, 0, 240, (u16*)sAcceptedSerialNos);
 }
 
-void sub_80F8D14(void)
+void LinkRfu_StopManagerBeforeEnteringChat(void)
 {
-    sub_80FD760(FALSE);
+    rfu_LMAN_stopManager(FALSE);
```

Comment:
`When 0 is specified for the u8 forced_stop_and_rfu_reset argument here, the user can continue with a new connection operation` so `FALSE` doesn't really make sense here